libs/SalesforceSDK/build.gradle

```diff
@@ -3,15 +3,12 @@ apply plugin: 'com.android.library'
 
 dependencies {
   compile 'com.google.guava:guava:18.0'
-  compile 'org.apache.httpcomponents:httpmime:4.3.2'
-  compile 'org.apache.james:apache-mime4j:0.7.2'
-  compile 'com.android.volley:volley:1.0.0'
+  compile 'com.squareup.okhttp3:okhttp:3.2.0'
```

Comment:
It's here: https://bintray.com/bintray/jcenter/com.squareup.okhttp3%3Aparent/view